keybert/backend/_spacy.py

```diff
@@ -74,21 +73,14 @@ def embed(self, documents: List[str], verbose: bool = False) -> np.ndarray:
             Document/words embeddings with shape (n, m) with `n` documents/words
             that each have an embeddings size of `m`
         """
-
         # Extract embeddings from a transformer model
         if "transformer" in self.embedding_model.component_names:
             embeddings = []
             for doc in tqdm(documents, position=0, leave=True, disable=not verbose):
                 try:
-                    embedding = (
-                        self.embedding_model(doc)._.trf_data.tensors[-1][0].tolist()
-                    )
-                except:
-                    embedding = (
-                        self.embedding_model("An empty document")
-                        ._.trf_data.tensors[-1][0]
-                        .tolist()
-                    )
+                    embedding = self.embedding_model(doc)._.trf_data.tensors[-1][0].tolist()
+                except:  # noqa: E722
```

Comment:
This was flagged as E722 (bare except).

What is the except clause catching here?

Reply:
It's been a while but I believe there were several issues here. Sometimes it couldn't find any embedding and depending on the version it would spit out different types of errors (also depending on the underlying embedding model type). I believe that's why I kept it bare.

Reply:
Alright, then it stays like that with the ignore?